### PR TITLE
Update reward readme

### LINKED ISSUE #1008: reward/instructor/trainer.py needs to handle default values of config vars in training_conf (loaded from .yml file)
I have tried to run the training "python trainer.py configs/electra-base-dis-webgpt.yml" and ran into an issue of
"Keyerror: scheduler" at line 140
         lr_scheduler_type=training_conf["scheduler"]
Further inspection revealed that it is caused by missing "scheduler" from "configs/electra-base-dis-webgpt.yml". As a result, the key to index the training_conf data structure does not exist (the "scheduler" in this case). The current version of trainer.py simply throws an error in that case.  I suggest that we modify the behavior of trainer.py so that it uses a default value for a certain config var if it is not defined in the .yml file. This should apply to all such config vars, and "scheduler" happens to be the one of them I just spotted

### PR DESCRIPTION
**Pull request by jackapbutler**
* fixes #1008 by following the most recent advice in that thread

This small PR just adds a `dummy.yml` file containing all possible configuration options in the reward model training as well as updating the `README.md` so people are aware they need to set this up before they run the `trainer.py`.

### CODE REVIEW

**Review by sanagno — APPROVED** [review]
Thanks!

**Comment by henryj18** [comment]
Nice job and kudos to your guys, appreciate your hard work!

On retrospect, could we streamline the process by not opening a new issue, but instead placing the people working on 1438 into the team to fix 1008? That can help reduce the complexity of managing multiple github issues on the exact same issue, and help to save repeated work. I was assigned 1008 earlier, and got notified that 1438 fixed 1008 only when 1438 was done and merged. As a result, the work I have done on 1008 and was just about to push into github become repeated. It would be great if we can put into place some process to avoid such repeated work in the future.

**Comment by sanagno** [comment]
That is a great point @henryj18, and apologies for the pure communication. Feel free to incorporate your work into the project now. We are in a stage where we need to move fast, and minor issues can be resolved faster. We can try to be more careful in the future!

**Comment by henryj18** [comment]
@sanagno Thanks very for your update, appreciate that. Since you guys have fixed the issue, there is no need for me to submit further work unless I find something new. I understand that you guys were under the pressure to close issues fast. Cordially look forward to some potential future collaborations.

**Comment by jackapbutler** [comment]
Hey @henryj18 apologies I just scanned through the thread and didn’t realise it was assigned to you. I was looking at similar files while investigating #397 and just made the change on the fly. I’ll be mindful in the future to check back on the issues for who’s assigned 🙂

**Comment by henryj18** [comment]
@jackapbutler Thanks for your update. No need to apologize, as I said above "I understand that you guys were under the pressure to close issues fast. Cordially look forward to some potential future collaborations." 🙂
